### PR TITLE
Add quote

### PR DESCRIPTION
**Pull request by s-Hale**
(also a whitespace fix automatically on save.)

## ✅ New Quote Checklist
- [x] The quote(s) added are in alphabetical order by author's first name and not ❌ duplicates.
- [x] The quote(s) added are about **learning a foreign language**.
- [x] The sentence attribute does not have inner quotation marks, and is ended with a period.
- [x] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

### CODE REVIEW

File: server/data/quotes.json [hunk 1]
```diff
@@ -315,6 +315,9 @@
     "author": "Ingrid Bengis",
     "sentence": "For me, words are a form of action, capable of influencing change."
   },
+  {"author": "J.R.R. Tolkien",
+  "sentence": "It is wrong to call the language you grow up speaking your native language. It is not. It is your first learnt language."
+  },
```
Comment (Kvaibhav01) [inline server/data/quotes.json:320]: Please format the JSON code. You can use this [tool](https://jsonformatter.curiousconcept.com).

Make sure to [check the linting](https://jsonlint.com).

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
Thanks for the PR! Just make the necessary changes.

File: server/data/quotes.json (hunk 1 above)
Comment (s-Hale) [reply server/data/quotes.json]: Have run it through the tool, I think it took off an extra space, though, so I re-added that. Should be correct now.

File: server/data/quotes.json [hunk 2]
```diff
@@ -312,13 +312,17 @@
...
-    "sentence": "For me, words are a form of action, capable of influencing change."
-  },
-  {
-    "author": "Jacob Ludwig Carl Grimm",
-    "sentence": "Our language is also our history."
-  },
+    "author":"Ingrid Bengis",
+    "sentence":"For me, words are a form of action, capable of influencing change."
+ },
+ {
```
Comment (kvharish) [inline server/data/quotes.json:318]: Could you please check the indentation here

**Review by kvharish — CHANGES_REQUESTED** [review]

File: server/data/quotes.json (hunk 2 above)
Comment (s-Hale) [reply server/data/quotes.json]: OK, have taken another look, seems some of the brackets went out of alignment? Weird. Formatting again has resulted in 13 changes. Hopefully just spaces.

**Review by jayehernandez — APPROVED** [review]
Thank you for contributing to Letra ☀️, @s-Hale!! Appreciate your contribution to this open source project. 🥳

---

Continue supporting Letra by:
- Leaving a star ⭐️ for this GitHub repository
- Downloading the free [Chrome extension](https://chrome.google.com/webstore/detail/letra/cjodkkjokggcaeacdhjliobekbnnmoio) and leaving a review
- Upvoting the [ProductHunt page](https://www.producthunt.com/posts/letra-2)

**Pull request merged**